api-docs.templ

```diff
@@ -0,0 +1,82 @@
+# API Documentation
+<a name="top"></a>
+
+{{range .Files}}
+{{$file_name := .Name}}
+<a name="{{.Name | anchor}}"></a>
+<p align="right"><a href="#top">Top</a></p>
+
+## {{.Name}}
+{{.Description}}
+
+{{range .Messages}}
+<a name="{{.FullName | anchor}}"></a>
+
+### {{.LongName}}
+{{.Description}}
+
+{{if .HasFields}}
```

Comment:
I notice the markdown generated from this template has a lot of extra line breaks. It's not a huge problem, by any means, but lots of markdown linters don't like that. fwiw, it's easy to avoid by favoring `{{- if ... -}}` and `{{- end -}}` over `{{ if ... }}` and `{{ end }}`. The dashes on the left and right eat up the extra whitespace to the left and right of the expression.

So, while this isn't a huge problem, it's one we can fix relatively easily, so I think we ought to.

Reply:
Sorry if I wasn't clear. Expanded use of `{{-` and `-}}` _throughout the entire template_ would eliminate many or all of the double line breaks in the rendered Markdown.